docs/getting-started/build-your-own-development-automation.md

```diff
@@ -217,6 +225,175 @@ Click on `Close` (you could also close the issue in GitHub if you prefer) and se
+
+That's the new generator created, now it's time to publish it so you can list and execute it using `@atomist` from your team's slack.
+
+### Publishing Your New Project Generator
+
+Similar to how you published your handlers project previously, this time you're going to publish your new generator rug archive. Execute the `rug publish` command from inside your generator's project directory:
+
+```
+> rug publish
+Resolving dependencies for yourorg:dotnetgettingstarted:0.1.0 ← local completed
+Invoking TypeScript Compiler on ts script sources
+  Created .atomist/editors/NewDotNetASPCoreProject.js.map
+  Created .atomist/editors/NewDotNetASPCoreProject.js
+Processing script sources completed
+Loading yourorg:dotnetgettingstarted:0.1.0 ← local into runtime completed
+  Created META-INF/maven/yourorg/dotnetgettingstarted/pom.xml
+  Created .atomist/manifest.yml
+  Created .atomist/metadata.json
+Generating archive metadata completed
+  Uploading yourorg/dotnetgettingstarted/0.1.0/dotnetgettingstarted-0.1.0.zip → t3v0s7ss2 (155 kb) succeeded
+  Uploading yourorg/dotnetgettingstarted/0.1.0/dotnetgettingstarted-0.1.0.pom → t3v0s7ss2 (659 bytes) succeeded
+  Uploading yourorg/dotnetgettingstarted/0.1.0/dotnetgettingstarted-0.1.0-metadata.json → t3v0s7ss2 (897 bytes) succeeded
+  Downloading yourorg/dotnetgettingstarted/maven-metadata.xml ← t3v0s7ss2 (394 bytes) succeeded
+  Uploading yourorg/dotnetgettingstarted/maven-metadata.xml → t3v0s7ss2 (346 bytes) succeeded
+Publishing archive into remote repository completed
+
+→ Archive
+  ~/code/src/github.com/yourorg/dotnetgettingstarted/.atomist/target/dotnetgettingstarted-0.1.0.zip (155 kb in 128 files)
+
+→ URL
+  https://atomist.jfrog.io/atomist/T3V0S7SS2/yourorg/dotnetgettingstarted/0.1.0/dotnetgettingstarted-0.1.0.zip
+```
+
+With your new generator's rug archive published it's now time to switch over to your team's slack again. Before you can see your new generator you will need to ask `@atomist` to refresh the rugs it is aware of by executing `@atomist refresh rugs`:
+
+<div class="ss-container">
+  <img src="../images/refreshing-rugs.png" alt="Refreshing the Rugs that Atomist is aware of" class="ss-small">
+</div>
+
+Now you can list the generators using `@atomist generators` and you should see your new generator in the list for your team:
+
+<div class="ss-container">
+  <img src="../images/new-generator-project-in-slack.png" alt="Your new generator all set to go in your team's slack" class="ss-small">
+</div>
+
+You've now converted a working project of your own to a new project generator to be used within Atomist! Go ahead and click the `Generate project` button in your team's slack and you can create a whole new project quickly and easily.
```

Comment:
Not sure about this as they _might_ not have done these actions previously if they skipped the Build a handler section.